src/cogent3/core/moltype.py

```diff
@@ -964,14 +964,14 @@ def is_valid(self, sequence):
         """Returns True if sequence contains no items that are not in self."""
         try:
             return self.first_invalid(sequence) is None
-        except:
+        except Exception:
```

Comment:
ignore changes to modules being deprecated, but do make those changes in the `new_<module name>`!